opal/tests/test_managers.py

```diff
@@ -205,3 +210,227 @@ def test_serialised_mine_tag_other_user(self):
             self.user, [self.episode]
         )
         self.assertEqual(as_dict[0]['tagging'][0], {'id': self.episode.id})
+
+
+class LookupListQuerysetTestCase(OpalTestCase):
+    def setUp(self, *args, **kwargs):
+        super(LookupListQuerysetTestCase, self).setUp(*args, **kwargs)
+        self.hat = self.create_model_and_synonym("Top Hat", "High Hat")
+
+    def create_synonym(self, instance, synonym):
+        ct = ContentType.objects.get_for_model(test_models.Hat)
+        return Synonym.objects.create(
+            name=synonym,
+            object_id=instance.id,
+            content_type=ct
+        )
+
+    def create_instance(self, name):
+        return test_models.Hat.objects.create(name=name)
```

Comment:
Having said I get why the methods are here, this indirection doesn't really seem that useful - it's only called once...